src/csharp/Grpc.IntegrationTesting/Control.cs

```diff
@@ -3847,7 +3847,9 @@ public sealed partial class ScenarioResultSummary : pb::IMessage<ScenarioResultS
     public const int QpsFieldNumber = 1;
     private double qps_;
     /// <summary>
-    /// Total number of operations per second over all clients.
+    /// Total number of operations per second over all clients. What is counted as 1 'operation' depends on the benchmark scenarios:
```

Comment:
Hmm I have no idea how these changes happened.

Reply:
Since these files were late regenerated someone added comments to the proto file. Proto comments turn into C# XML comments. It's _fine_.